server/service/scenarios/endpoints.js

```diff
@@ -3,17 +3,11 @@ const { asyncMiddleware } = require('../../util/api');
 const { reqScenario } = require('./middleware');
 const db = require('./db');
 
-exports.getScenario = asyncMiddleware(async function getScenarioAsync(
-    req,
-    res
-) {
+async function getScenarioAsync(req, res) {
```

Comment:
These scenario API refactors are great, but do we need them for your PR? If not, can we have them in another PR for ease of review/git history understanding?

Reply:
Sure, I can undo the work and re-submit.

Reply:
Thank you!

Reply:
Whoops, I forgot to do this change.